app/test/package/upload_test.dart

```diff
@@ -77,7 +82,9 @@ void main() {
           // Add one more byte than allowed.
           bigTarball.add([1]);
 
-          final sink = tarballStorage.bucket.write('tmp/$uploadId');
+          final bucket =
+              storageService.bucket(activeConfiguration.packageBucketName!);
+          final sink = bucket.write('tmp/$uploadId');
```

Comment:
We don't have this in `testWithProfile` the storage service is now exposed on localhost... So we can actually do a multipart upload.

Reply:
It was not exposed in the past, but I think this is exposed now, we just don't use it all the time (yet):
https://github.com/dart-lang/pub-dev/blob/master/app/lib/fake/server/fake_storage_server.dart#L37

Adding a TODO to migrate...